components/x-topic-search/src/TopicSearch.scss

```diff
@@ -38,6 +38,7 @@
 	max-width: none;
 	color: oColorsGetPaletteColor('white');
 	background: transparent;
+	height: 40px;
```

Comment:
Is it possible to set this instead with `line-height` (and possibly `padding`)? For accessibility, if users change their font size, it could mean that his hardcoded value may not be enough for larger fonts.

Using `line-height` should scale with the font changing.